ui/src/app/core/job-manager.service.ts

```diff
@@ -57,17 +57,25 @@ export class JobManagerService {
     return response;
   }
 
-  private getErrorTitle(response: any): string {
-    let json = response.json();
-    if ("title" in json && json["title"]) {
+  private respToJson(response: Response) {
+    try {
+      return response.json();
+    } catch (_) {
+      return {};
+    }
+  }
+
+  private getErrorTitle(response: Response): string {
+    const json = this.respToJson(response);
+    if (json["title"]) {
```

Comment:
Won't this throw a key error? I think you want if `if "title" in json`

Reply:
You're thinking of Python. JS will return `undefined` in this case.

Reply:
Yes I am sorry got confused 😕